articles/protocols/oauth2/oauth-state.md

```diff
@@ -1,24 +1,26 @@
 ---
-title: The State Parameter
+title: State Parameter
 description: Explains how to use the state parameter in authentication requests to help prevent CSRF attacks.
 ---
 
-# The State Parameter
+# State Parameter
 
 The `state` parameter is an authentication parameter used to help mitigate [CSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery).
```

Comment:
I think this needs more info. The "is an authentication parameter" is a bit generic, we don't even say what's the protocol that's using it. Or how this parameter actually helps against CSFR.

Maybe you could also use this as input https://community.auth0.com/t/usage-of-the-oauth-state-parameter/8266 (but write it in a more understandable format)